File: fix_css.py
import sys

with open('frontend/style.css', 'r', encoding='utf-8') as f:
    content = f.read()

# Fix zigzag
content = content.replace('''#chat-input {
    flex-grow: 1;
    background: transparent;
    border: none;
    color: var(--text-main);
    font-family: var(--font-main);
    font-size: 14px;
    padding: 8px 4px;
    outline: none;
    resize: none;
    overflow-y: auto;
    max-height: 120px;
    min-height: 38px;
    line-height: 1.4;
    align-self: center;
}''', '''#chat-input {
    flex-grow: 1;
    background: transparent;
    border: none;
    color: var(--text-main);
    font-family: var(--font-main);
    font-size: 14px;
    padding: 8px 4px;
    outline: none;
    resize: none;
    overflow-y: auto;
    max-height: 120px;
    min-height: 36px;
    line-height: 1.4;
    align-self: flex-end;
}''')

# Fix sidebar header
content = content.replace('''.sidebar-header {
    padding: 16px;
    background: rgba(0, 0, 0, 0.1);
    border-bottom: 1px solid var(--border-metal);
    display: flex;
    flex-direction: column;
    gap: 8px;
    height: 70px;
    box-sizing: border-box;
    flex-shrink: 0;
}''', '''.sidebar-header {
    padding: 16px;
    background: transparent;
    border-top: none;
    border-bottom: 1px solid var(--border-metal);
    display: flex;
    flex-direction: column;
    gap: 8px;
    height: 70px;
    box-sizing: border-box;
    flex-shrink: 0;
}''')

with open('frontend/style.css', 'w', encoding='utf-8') as f:
    f.write(content)

print("CSS fixed.")

File: fix_css.js
const fs = require('fs');
let css = fs.readFileSync('e:/AgentZero/usr/projects/skufia/frontend/style.css', 'utf8');
css = css.replace(/height: var\(--app-height\);/g, 'height: var(--app-height, 100dvh);')
         .replace(/max-height: var\(--app-height\);/g, 'max-height: var(--app-height, 100dvh);')
         .replace(/width: 100%; height: 100%;\s*z-index: 50;/g, 'right: 0; bottom: 0;\r\n        z-index: 50;');
if (!css.includes('.chat-messages::before')) {
    css = css.replace(/\.chat-messages \{\s*flex-grow: 1;/g, '.chat-messages::before {\r\n    content: \'\';\r\n    margin-top: auto;\r\n}\r\n\r\n.chat-messages {\r\n    flex-grow: 1;');
}
fs.writeFileSync('e:/AgentZero/usr/projects/skufia/frontend/style.css', css);
